css/order.css

```diff
@@ -0,0 +1,227 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* font-family: 'Montserrat', sans-serif;
+font-family: 'M PLUS 1p', sans-serif;
+font-family: 'Work Sans', sans-serif; */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  outline: 0;
+  border: none;
+}
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  text-align: center;
+  color: #666666;
+}
+
+.form-section {
+  width: 100%;
+  display: flex;
+}
+
+.image-container {
+  width: 50%;
+}
+
+.form-image {
+  height: auto;
+  width: 100%;
+}
+
+.form-container {
+  width: 50%;
+  font-family: "M PLUS 1p", sans-serif;
+  align-items: flex-start;
+  text-align: start;
+  padding: 4rem 6rem;
+}
+
+.form-container h1 {
+  color: #de6e49;
+  font-size: 2.1rem;
+  font-weight: 600;
+}
+
+.form-container input {
+  border: 1px solid grey;
+  border-radius: 4px;
+}
+
+.borders label {
+  display: flex;
+  margin-top: 18px;
+}
+
+.oneline-input {
+  display: flex;
+  justify-content: space-between;
+}
+
+.required {
+  margin-left: 3px;
+}
+input[type="text"],
+input[type="email"],
+select {
+  width: 170px;
+  height: 35px;
+  padding: 3px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid rgb(104, 101, 101);
+  border-radius: 4px;
+  box-sizing: border-box;
+  vertical-align: middle;
+}
+
+input.address {
+  width: 100%;
+}
+
+select {
+  width: 210px;
+  font-style: italic;
+  color: rgb(110, 107, 107);
+}
+
+input.postcode {
+  width: 130px;
+}
+
+.color {
+  display: flex;
+  text-align: center;
+}
```

Comment:
Try and give your class names a name that represent what they target or what they do. This class doesn't apply any colour! :)